README.md

```diff
@@ -1,3 +1,30 @@
+# Announcement
+
+There are two known issues caused by the upstream tool. Please follow the temporary solutions below to resolve them:
+
+### Issue #1
+`comm_size` is falsely set to zero due to mismatched `protobuf` types (`uint64` vs `int64`).
```

Comment:
This issue typically arises from a type mismatch between the trace and Chakra. For instance, it can occur when ETFeeder attempts to read Chakra traces that are encoded with a different type than expected. Ensuring that the trace format matches Chakra’s encoding type should resolve the error.

Reply:
Let's add this sentence before: 
"Simulation appears as if communication is not happening. This is because .."